ONSdigital/eq-author-app
# Add function to get datatype if it is missing from entity

### What is the context of this PR?
No comma is showing up when piping a currency answer into a question for some of the older questionnaires. This is because the \<span\> entity is missing the "data-type" property.

Added a function that will check if the data-type is missing from \<span\>. If data-type is missing, the function will get the data-type from the questionnaireJson and insert it into the variable.

Added a test for when a \<span\> is missing its data-type property and the data-type can be accessed through the questionnaireJson. Code should return a span with the correct data-type inserted.

### How to review 
1. Create a new questionnaire with a currency answer piped into another question. 
2. Open the dynamodb-admin and remove the "data-piped" property from the \<span\> entity. 
3. View the questionnaire and see that the currency answer is correctly piped in as a "Currency" data type.
